modules/spo-log-enricher-app-trace.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-advanced.adoc
+
+:_content-type: PROCEDURE
+[id="spo-log-enricher-app-trace_{context}"]
+= Using the log enricher to trace an application
+
+You can use the Security Profiles Operator log enricher to trace an application.
+
+.Procedure
+
+. To trace an application, create a `SeccompProfile` logging profile:
++
+[source,yaml]
+----
+apiVersion: security-profiles-operator.x-k8s.io/v1beta1
+kind: SeccompProfile
+metadata:
+  name: log
+  namespace: default
+spec:
+  defaultAction: SCMP_ACT_LOG
+----
+
+. Create a pod object to use the profile:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: log-pod
+spec:
+  securityContext:
+    seccompProfile:
+      type: Localhost
+      localhostProfile: operator/default/log.json
+  containers:
+    - name: log-container
+      image: nginx
```

Comment:
The same issue:
quay.io/security-profiles-operator/test-nginx-unprivileged:1.21 would be a better example